rust/automerge/src/op_tree.rs

```diff
@@ -27,6 +27,7 @@ pub(crate) struct OpTree {
     pub(crate) objtype: ObjType,
     /// The id of the parent object, root has no parent.
     pub(crate) parent: Option<ObjId>,
+    pub(crate) last_insert: Option<(usize, usize)>,
```

Comment:
Can we add a doc comment explaining this field? It would be useful to know when it needs to be cleared and what it's used for.